amti/clis/review.py

```diff
@@ -28,6 +28,11 @@
     '--approve-all', '-a',
     is_flag=True,
     help="Approve all submissions.")
+@click.option(
+    '--mark-file', '-m',
+    type=str,
+    default=None,
+    help="Name of the marked files to be saved or empty to output to stdout")
```

Comment:
Using click's [Path type](https://click.palletsprojects.com/en/7.x/parameters/#parameter-types) rather than `str` would enable click to immediately validate the option, and present a more helpful error message:

```python
@click.option(
    '--mark-file', '-m',
    type=click.Path(
        exists=False,
        file_okay=True,
        dir_okay=False,
        writable=True,
        allow_dash=True),
    default='-',
    help='The path to the file in which to save the marked assignments. Defaults to STDOUT.')
```

As an added benefit, we can handle defaulting to STDOUT here, simplifying the logic and API in the backend: